src/Altinn.Broker/Models/FileInitializeExt.cs

```diff
@@ -0,0 +1,46 @@
+using System.Text.Json.Serialization;
+
+namespace Altinn.Broker.Models
+{
+    /// <summary>
+    /// Entity containing BrokerFile initialization data.
+    /// </summary>
+    public class FileInitalizeExt
+    {
+        /// <summary>
+        /// Gets or sets the original filename
+        /// </summary>
+        [JsonPropertyName("filename")]
+        public string FileName { get; set; } = string.Empty;
+
+        /// <summary>
+        /// Gets or sets the senders file reference. Used by senders and receivers to identify specific file using external identification methods.
+        /// </summary>
+        [JsonPropertyName("sendersFileReference")]
+        public string SendersFileReference { get; set; } = string.Empty;
+
+        /// <summary>
+        /// Gets or sets checksum for file data.
+        /// </summary>
+        [JsonPropertyName("checksum")]
+        public string Checksum { get; set; } = string.Empty;
+
+        /// <summary>
+        /// Gets or sets the sender of the broker file.
+        /// </summary>
+        [JsonPropertyName("sender")]
+        public string Sender { get; set; } = string.Empty;
+
+        /// <summary>
+        /// Gets or sets the recipients of the broker file.
+        /// </summary>
+        [JsonPropertyName("recipients")]
+        public List<string> Recipients { get; set; } = new List<string>();
+
+        /// <summary>
+        /// Gets or sets the properties field.
+        /// </summary>
+        [JsonPropertyName("metadata")]
+        public Dictionary<string, string> Metadata { get; set; } = new Dictionary<string, string>();
```

Comment:
Consider changing the property named "MetaData" to "PropertyList" as this is a less generic name and reflects the naming in the legacy solution for the same purpose. This also avoids confusion with the property "Metadata" in FileInitializeAndUploadExt.